build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# check if virtualenv already exists
+if [ -d "$HOME/ubportsdocsenv" ]; then
+  echo "Build environment found"
+  . ~/ubportsdocsenv/bin/activate
+else
+  echo "No build environment found. Creating..."
```

Comment:
Maybe say where we're putting the doc so that no one gets scared? Also consider adding some hashmarks or something so that our status echos stand out from regular command output